File: Java/Handphone.java
public class Handphone extends SpecHandphone {
    private String manufaktur;
    private String series;
    private String tipeOS;

    public Handphone() {}

    public Handphone(String idProduk, String nama, int harga,
                     String processor, int ram, int storage, int battery,
                     String manufaktur, String series, String tipeOS) {
        super(idProduk, nama, harga, processor, ram, storage, battery);
        this.manufaktur = manufaktur;
        this.series = series;
        this.tipeOS = tipeOS;
    }

    public String getManufaktur() { return manufaktur; }
    public void setManufaktur(String manufaktur) { this.manufaktur = manufaktur; }

    public String getSeries() { return series; }
    public void setSeries(String series) { this.series = series; }

    public String getTipeOS() { return tipeOS; }
    public void setTipeOS(String tipeOS) { this.tipeOS = tipeOS; }
}

File: Python/Handphone.py
from SpecHandphone import SpecHandphone

class Handphone(SpecHandphone):
    def __init__(self, id_produk="", nama="", harga=0,
                 processor="", ram=0, storage=0, battery=0,
                 manufaktur="", series="", tipe_os=""):
        super().__init__(id_produk, nama, harga, processor, ram, storage, battery)
        self.__manufaktur = manufaktur
        self.__series = series
        self.__tipe_os = tipe_os

    def get_manufaktur(self): return self.__manufaktur
    def set_manufaktur(self, manufaktur): self.__manufaktur = manufaktur

    def get_series(self): return self.__series
    def set_series(self, series): self.__series = series

    def get_tipe_os(self): return self.__tipe_os
    def set_tipe_os(self, tipe_os): self.__tipe_os = tipe_os
